docs/contributing/contributing-code/contributing-code-tests/tests-images-pushtoghcr.md

```diff
@@ -0,0 +1,64 @@
+
+## Using Standard Images in Tests
+
+When writing tests for our applications, it's common to use standard images for resources like Redis, Mongo, and others. By default, specifying an image with `redis:latest` or similar tags will pull the image from Docker Hub. This behavior is due to Docker's default configuration, which automatically searches the Docker Hub registry for images when no other registry is specified. 
+
+ We are transitioning to using images hosted in our public GitHub Container Registry (GHCR) repository. This change ensures more reliable access to the images needed for our tests and will help reduce potential issues caused by external dependencies.
+
+### New Image Pull Guideline
+
+Moving forward, we will pull standard test images from our public GHCR repository instead of Docker Hub. 
+
+For example, instead of using:
+```yaml
+image: 'redis:latest'
+```
+We should now use:
+```yaml
+image: ghcr.io/radius-project/mirror/redis:latest
+```
+This change applies to all standard images previously pulled from Docker Hub. Please update any new tests accordingly.
+
+### Adding Images to GHCR Repository
+
```

Comment:
Can we add what roles you need to have to update these permissions? i.e. approvers can push to GHCR, maintainers can change package permissions (not sure if these are accurate but just examples)

Reply:
In our meeting, it was decided that maintainers should have admin access while it's read only for all others. We can bring it up again if this needs to be reconsidered.